docs/_interface/consensus-clients.md

```diff
@@ -0,0 +1,149 @@
+intend to continue to run full nodes after The Merge, it is sensible to start running a consensus client now, 
+so that The Merge can happen smoothly. There are five consensus clients available, all of which connect to Geth in the same way. 
+
+This page will outline how Geth can be set up with a consensus client in advance of The Merge (or to interact with an alread-merged testnet).
+
+
+## Configuring Geth
+
+Geth can be downloaded and installed according to the instructions on the 
+[Installing Geth](/docs/install-and-build/installing-geth) page. In order to connect to a consensus client,
+Geth must expose a port for the inter-client RPC connection. 
+
+The RPC connection must be authenticated using a `jwtsecret` file. This is created and saved 
+to `<datadir>/geth/jwtsecret` by default but can also be created and saved to a custom location or it can be
+self-generated and provided to Geth by passing the file path to `--authrpc.jwtsecret`. The `jwtsecret` file 
+is required by both Geth and the consensus client.
+
+The authorization must then be applied to a specific address/port. This is achievd by passing an address to
+`--authrpc.addr` and a port number to `--authrpc.port`. It is also safe to provide either `localhost` or a wildcard
+`*` to `--authrpc.vhosts` so that incoming requests from virtual hosts are accepted by Geth because it only 
+applies to the port authenticated using `jwtsecret`. 
+
+The Merge itself will be triggered using a terminal total difficulty (TTD). The specific value for the TTD has not yet
+been decided. When it is decided,Geth needs to know what it is in order to merge successfully. This requires restarting
+Geth with the `--override.terminaltotaldiffilculty` command, passing the correct value as an argument.
+
+A complete command to start Geth so that it can connect to a consensus client looks as follows:
+
+```shell
+geth --authrpc.addr localhost --authrpc.port 8551 --authrpc.vhosts localhost --authrpc.jwtsecret /tmp/jwtsecret
+```
+
+
+## Consensus clients
+
+There are currently five consensus clients that can be run alongside Geth. These are:
+
+[Teku](https://pegasys.tech/teku): written in Java
+ 
+[Nimbus](https://nimbus.team/): written in Nim
+ 
+[Lighthouse](https://lighthouse-book.sigmaprime.io/): written in Rust
+ 
+[Lodestar](https://lodestar.chainsafe.io/): written in Typescript
+ 
+[Prysm](https://docs.prylabs.network/docs/getting-started/): written in Go
+
+There is currently a [client diversity][client-div-link] issue where a large dominance of Prysm clients poses a risk 
+to the health of the network. In response to the initial drive to even out the client diversity many Prysm nodes switched 
+to Lighthouse to the extent that it now also has a problematic market share. It is therefore recommended to use a minority 
+client such as Teku, Nimbus or Lodestar (although users should be aware that Lodestar is currently awaiting a third-party audit). 
```

Comment:
To be honest, I'm not sure if we should even link to Lodestar as the staking launchpad also doesn't: https://launchpad.ethereum.org/

Reply:
yea on second thought it is better to be consistent with other EF pages - launchpad are waiting on lodestar's audit I believe. Thanks.